### DISCUSSION TITLE
help w/ installation?

### DISCUSSION THREAD

**Discussion by presmihaylov** [CopilotC-Nvim/CopilotChat.nvim#57] [Q&A] [answered]
I have neither lazy vim or vim plug setup, and the manual installation instructions don't work. 

When using cp, I get that `--backup` flag doesn't exist (I'm on mac) and if I omit it and follow rest of instructions I see this anytime I run a `CopilotChat` command:
<img width="1178" alt="image" src="https://github.com/CopilotC-Nvim/CopilotChat.nvim/assets/7642330/cfd7ec93-e651-4cfe-a2e8-0c6ec7d38fe6">

I use pathogen but there are no instructions for it.

**Comment by neutrinoA4** [comment, marked as answer]
@preslavmihaylov 
I guessed that you weren't calling the setup function.

If you are using a plugin manager other than the one mentioned in the README, try calling the setup function in your lua file as follows.
```
local copilot_chat = require("CopilotCaht")
copilot_chat.setup({})
```

**Reply by presmihaylov** [reply]
Thanks, that did the trick!

**Comment by jellydn** [comment]
Hi @preslavmihaylov 
Could you try to call `setup` function? If that doesn't work, could you try to add the below to your config?

```lua
let g:copilot_chat_disable_separators = "no"
let g:copilot_chat_proxy = ""
lua _COPILOT_CHAT_GLOBAL_CONFIG = _COPILOT_CHAT_GLOBAL_CONFIG or {}
let g:copilot_chat_user_prompts = {
  Explain = 'Explain how it works.',
  Tests = 'Briefly explain how selected code works then generate unit tests.',
}
let g:copilot_chat_show_help = 'yes'
let copilot_chat_disable_separators = 'no'
let copilot_chat_hide_system_prompt = 'yes' 
let copilot_chat_language = '' 
```
